changelogs/unreleased/gh-6253-iproto-id.md

```diff
@@ -0,0 +1,5 @@
+## feature/core
+
+* Extended the network protocol (IPROTO) with a new request type (`IPROTO_ID`)
+  that is supposed to be used for exchanging sets of supported features between
+  server and client (gh-6253).
```

Comment:
You're right, we don't really need them, but I'd like to have some non-empty initial feature set, because it's useful for testing and documentation. These two features were added recently (in beta) and they were major recent extension of the protocol so I think it should be fine to add them. I guess we could add `IPROTO_CHUNK` to the feature set as well.